IAGrim/UI/Controller/SearchController.cs

```diff
@@ -166,41 +156,34 @@ AugmentationItemRepo augmentationItemRepo
 
             List<PlayerHeldItem> items = new List<PlayerHeldItem>();
             items.AddRange(_playerItemDao.SearchForItems(query));
-           
 
             // This specific filter was easiest to add after the actual search
             // Obs: the "duplicates only" search only works when merging duplicates
             if (duplicatesOnly) {
                 items = items.Where(m => m.Count > 1).ToList();
             }
-
             
             personalCount = items.Sum(i => i.Count);
 
             if (includeBuddyItems && !query.SocketedOnly) {
                 AddBuddyItems(items, query, out message);
             }
-            else {
-                if (personalCount == 0)
-                    message = "No matching items found";
-                else {
-                    message = string.Empty;
-                }
+            else
+            {
+                // TODO: Translation
```

Comment:
Trivial to move the string to the EnglishLanguage class, and fetch this one as a tag.

If not, this will just end up as a request from the Japanese community in a week.

Reply:
Yes I know. I'll do that in hopefully less than a week along with other untranslated strings. :)